{
  "Header": {
    "Original": "CVE-2021-23186 (A sandboxing issue in Odoo Community 15.0 and earlier and Odoo Enterpr ...)",
    "Line": 110888,
    "ID": "CVE-2021-23186",
    "Description": "(A sandboxing issue in Odoo Community 15.0 and earlier and Odoo Enterpr ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-5399-1}",
      "Line": 110889,
      "Type": "xref",
      "Bugs": [
        "DSA-5399-1"
      ]
    },
    {
      "Original": "- odoo \u003cunfixed\u003e (bug #1035953)",
      "Line": 110890,
      "Type": "package",
      "Package": "odoo",
      "Kind": "unfixed",
      "BugNo": 1035953
    },
    {
      "Original": "NOTE: https://github.com/odoo/odoo/issues/107688",
      "Line": 110891,
      "Type": "NOTE",
      "Description": "https://github.com/odoo/odoo/issues/107688"
    },
    {
      "Original": "NOTE: https://github.com/odoo/odoo/commit/c1d6d4a1d9148275213c7f3c286658366df03bd7",
      "Line": 110892,
      "Type": "NOTE",
      "Description": "https://github.com/odoo/odoo/commit/c1d6d4a1d9148275213c7f3c286658366df03bd7"
    }
  ]
}